<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaixaTextoWidget</class>
 <widget class="QWidget" name="CaixaTextoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="caixatexto_grid">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="negrito_chk">
       <property name="text">
        <string>Negrito</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="italico_chk">
       <property name="text">
        <string>It√°lico</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>365</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QPlainTextEdit" name="texto_txt">
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="sel_cor_tb">
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>23</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
